### PR TITLE
fix: disable ssl_session_tickets by default

### PR DESCRIPTION
**Pull request by Miss-you**
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

**Comment by Miss-you** [comment]
#2638

**Comment by Miss-you** [comment]
another bug（maybe）：https://github.com/apache/apisix/issues/2660

File: conf/config-default.yaml [hunk 1]
```diff
@@ -106,6 +106,7 @@ apisix:
                                                 # with external services (e.g. etcd)
     ssl_protocols: "TLSv1.2 TLSv1.3"
     ssl_ciphers: "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384"
+    ssl_session_tickets: false              #  disable ssl_session_tickets by default for 'ssl_session_tickets' would make Perfect Forward Secrecy useless.
```
Comment (spacewander) [inline conf/config-default.yaml:109]: Better to add the link in the comment, so the user can find more details.

Comment (Miss-you) [reply conf/config-default.yaml:109]: done

**Review by spacewander — APPROVED** [review]

File: .travis/apisix_cli_test.sh [hunk 2]
```diff
@@ -447,17 +447,17 @@ git checkout conf/config.yaml
 
 make init
 
-count=`grep "ssl_session_tickets off;" conf/nginx.conf || true`
-if [ $count -ne 1 ]; then
+count=`grep "ssl_session_tickets off;" conf/nginx.conf | wc -l `
```
Comment (membphis) [inline .travis/apisix_cli_test.sh]: we can use this way: https://github.com/apache/apisix/blob/8ed24711a6c659fa8040310a24e86091d104a699/.travis/apisix_cli_test.sh#L122

Comment (Miss-you) [reply .travis/apisix_cli_test.sh]: done. `grep` -> `grep -c`

**Review by membphis — APPROVED** [review]

**Pull request merged**